smartbms.py

```diff
@@ -31,59 +31,56 @@ class BatteryChemistry:
     NCA = 4
 
 class MAFilter:
-    def __init__(
-        self,
-        filter_size,
-        initial_value):
-        self.buffer = [initial_value]*filter_size
+    '''
+    Add a doc string
+    '''
+    def __init__(self, filter_size: int, initial_value: bool):
```

Comment:
Filter_size is inded an int, but initial_value can have multiple types

Reply:
```suggestion
   from typing import Union
    def __init__(self, filter_size: int, initial_value: Union[bool, float, int]):
```

This is possible if the python version is at least 3.5 we can use Union (see above).  Would this cover all the possible types?